pkg/operator/ceph/operator.go

```diff
@@ -101,6 +102,22 @@ func (o *Operator) cleanup(stopCh chan struct{}) {
 	o.clusterController.StopWatch()
 }
 
+func (o *Operator) updateOperatorLogLevel() error {
+	rookLogLevel, err := k8sutil.GetOperatorSetting(o.context.Clientset, opcontroller.OperatorSettingConfigMapName, "ROOK_LOG_LEVEL", "INFO")
+	if err != nil {
+		logger.Warningf("failed to load ROOK_LOG_LEVEL. Defaulting to INFO. %v", err)
+		rookLogLevel = "INFO"
+	}
+
+	logLevel, err := capnslog.ParseLevel(rookLogLevel)
+	if err != nil {
+		return errors.Wrapf(err, "failed to load ROOK_LOG_LEVEL %q.", logLevel)
```

Comment:
```suggestion
		return errors.Wrapf(err, "failed to load ROOK_LOG_LEVEL %q.", rookLogLevel)
```

Shouldn't this be `rookLogLevel`? On a failure `logLevel` would be empty right?

Reply:
it should be `rookLogLevel` but I see it return `return CRITICAL, errors.New("couldn't parse log level " + s)`, changed.